grommet/grommet
# Replace hpe temp theme from AllComponents Story

#### What does this PR do?
The hpe temporary theme was replace by the official 'grommet-hpe-theme'
#### Where should the reviewer start?

#### What testing has been done on this PR?
The `AllComponents` Story was checked 
#### How should this be manually tested?
Open the storybook locally and check if the `AllComponents` Story is working properly
#### Any background context you want to provide?
No
#### What are the relevant issues?
None
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/20492786/87585513-91758600-c6b5-11ea-8f21-944bc314b4af.png)

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
No
